rules/Php70/Rector/FuncCall/MultiDirnameRector.php

```diff
@@ -56,12 +56,14 @@ public function refactor(Node $node): ?Node
         $activeFuncCallNode = $node;
         $lastFuncCallNode = $node;
 
+        $shouldUpdate = false;
```

Comment:
We check that there is at least one nested dirname call, if not the rule is not applied